pkg/state/steps_uki.go

```diff
@@ -290,10 +303,11 @@ func (s *State) UkiPivotToSysroot(g *herd.Graph) error {
 				internalUtils.DropToEmergencyShell()
 			}
 
-			output, pcrErr := internalUtils.CommandWithPath("/usr/lib/systemd/systemd-pcrphase --graceful enter-initrd")
+			extension := "enter-initrd"
+			output, pcrErr := UKIExtendPCR(extension)
 			if pcrErr != nil {
-				internalUtils.Log.Err(pcrErr).Msg("running systemd-pcrphase")
-				internalUtils.Log.Debug().Str("out", output).Msg("systemd-pcrphase enter-initrd")
+				internalUtils.Log.Err(pcrErr).Msg("running systemd-extend")
+				internalUtils.Log.Debug().Str("out", output).Msg(fmt.Sprintf("systemd-extend %s", extension))
```

Comment:
For this is better to use structured logs. I.e.

Log.Debug().Str("extension": extension).Str("output": output.....


As that is then easily ingested into other log structured platforms for easy of filtering and aggregation and so on